mozilla-release/toolkit/content/widgets/browser.xml

```diff
@@ -1301,6 +1301,9 @@
             aOtherBrowser[field] = ourFieldValues[field];
           }
 
+          // This MUST be reset in order to reflect docShell swap.
+          this._loadContext = null;
```

Comment:
This is the second critical change.
Cached value prevented us from seeing real privateness status of a swapped tab.